File: bee_1564.py
def main():
    while True:
        # Enquanto não tiver acabado o arquivo:
        try:
            n = int(input())
            # Se não houver reclamações
            if (n == 0):
                print("vai ter copa!")
            # Se reclamar, vai ter duas
            else:
                print("vai ter duas!")
        except EOFError:
            # Encerrar while loop
            break

main()

File: bee_1564.c
#include <stdio.h>

int main() {
    while (1) {
        // Enquanto não tiver acabado o arquivo:
        int n;
        if (scanf("%d", &n) == EOF) {
            // Encerrar while loop
            break;
        }
        // Se não houver reclamações
        if (n == 0) {
            printf("vai ter copa!\n");
        }
        // Se reclamar, vai ter duas
        else {
            printf("vai ter duas!\n");
        }
    }

    return 0;
}